src/WindowsTimezone.php

```diff
@@ -0,0 +1,117 @@
+        'Arab Standard Time'                        => 'Asia/Riyadh',
+        'E. Africa Standard Time'                   => 'Africa/Nairobi',
+        'Arabic Standard Time'                      => 'Asia/Baghdad',
+        'Iran Standard Time'                        => 'Asia/Tehran',
+        'Arabian Standard Time'                     => 'Asia/Dubai',
+        'Caucasus Standard Time'                    => 'Asia/Yerevan',
+        'Transitional Islamic State of Afghanistan Standard Time' => 'Asia/Kabul',
+        'Ekaterinburg Standard Time'                => 'Asia/Yekaterinburg',
+        'West Asia Standard Time'                   => 'Asia/Tashkent',
+        'India Standard Time'                       => 'Asia/Colombo',
+        'Nepal Standard Time'                       => 'Asia/Colombo', //fallback on India Standard Time because can be absent on outdated database
+        'Central Asia Standard Time'                => 'Asia/Almaty',
+        'Sri Lanka Standard Time'                   => 'Asia/Colombo',
+        'N. Central Asia Standard Time'             => 'Asia/Novosibirsk',
+        'Myanmar Standard Time'                     => 'Asia/Rangoon',
+        'S.E. Asia Standard Time'                   => 'Asia/Bangkok',
+        'North Asia Standard Time'                  => 'Asia/Krasnoyarsk',
+        'China Standard Time'                       => 'Asia/Shanghai',
+        'Singapore Standard Time'                   => 'Asia/Singapore',
+        'Taipei Standard Time'                      => 'Asia/Taipei',
+        'W. Australia Standard Time'                => 'Australia/Perth',
+        'North Asia East Standard Time'             => 'Asia/Irkutsk',
+        'Korea Standard Time'                       => 'Asia/Seoul',
+        'Tokyo Standard Time'                       => 'Asia/Tokyo',
+        'Yakutsk Standard Time'                     => 'Asia/Yakutsk',
+        'A.U.S. Central Standard Time'              => 'Australia/Darwin',
+        'Cen. Australia Standard Time'              => 'Australia/Adelaide',
+        'A.U.S. Eastern Standard Time'              => 'Australia/Sydney',
+        'E. Australia Standard Time'                => 'Australia/Brisbane',
+        'Tasmania Standard Time'                    => 'Australia/Hobart',
+        'Vladivostok Standard Time'                 => 'Asia/Vladivostok',
+        'West Pacific Standard Time'                => 'Pacific/Port_Moresby',
+        'Central Pacific Standard Time'             => 'Pacific/Guadalcanal',
+        'Fiji Islands Standard Time'                => 'Pacific/Fiji',
+        'New Zealand Standard Time'                 => 'Pacific/Auckland',
+        'Tonga Standard Time'                       => 'Pacific/Tongatapu'
+    ];
+
+    /**
+     * return a valid IANA timezone
+     *
+     * @return string
+     */
+    public function getTimezone($timezone)
+    {
+        if (isset($this->timezone[$timezone])) {
+            return $this->timezone[$timezone];
+        }
```

Comment:
It's not really easy to test if it is valid. You can't just instanciate a new `\DateTimezone` and wrap it in a `try/catch`. It can works but you can't retrieve after the offset or the location.